egs2/README.md

```diff
@@ -95,6 +95,7 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 | swbd_sentiment          | Speech Sentiment Annotations                                                                                                     | SLU                     | ENG                  | https://catalog.ldc.upenn.edu/LDC2020T14                                                                     |              |
 | talromur                | Talromur: A large Icelandic TTS corpus                                                                                                        | TTS                     | ISL                  | https://repository.clarin.is/repository/xmlui/handle/20.500.12537/104, https://aclanthology.org/2021.nodalida-main.50.pdf                |              |
 | tedlium2                | TED-LIUM corpus release 2                                                                                                        | ASR                     | ENG                  | https://www.openslr.org/19/, http://www.lrec-conf.org/proceedings/lrec2014/pdf/1104_Paper.pdf                |              |
+| tedx_spanish_corpus_openslr67                | TEDx Spanish Corpus                                                                                                        | ASR                     | SPA                 | https://www.openslr.org/67/                |              |
```

Comment:
`tedx_spanish_corpus_openslr67` sounds too long.
How about `tedx_spanish_openslr67`

Reply:
can do; made the corresponding changes